src/commands/create-manifest/create-manifest.ts

```diff
@@ -56,64 +65,60 @@ export const commandCreateManifest = command({
 
         // Store the list of files to move in a bucket rather than the ARGO parameters
         if (actionLocation) {
-          const targetLocation = fsa.join(actionLocation, `actions/manifest-${targetHash}.json`);
+          const targetLocation = fsa.join(actionLocation.href, `actions/manifest-${targetHash}.json`);
           const targetAction: ActionCopy = { action: 'copy', parameters: { manifest: current } };
           await fsa.write(targetLocation, JSON.stringify(targetAction));
-          outputCopy.push(targetLocation);
+          outputCopy.push(new URL(targetLocation));
         } else {
-          outputCopy.push(gzipSync(outBuf).toString('base64url'));
+          outputCopy.push(new URL(gzipSync(outBuf).toString('base64url')));
```

Comment:
`base64url` is never a URL

Reply:
OIC, [base64url encoding is used to create URL-safe strings from binary data](https://github.com/nodejs/node/issues/26512). Fixed.